joomla/joomla-cms
# [4.0] Adding some quickstart in default and alternate menu presets

### Summary of Changes
Quickstarts are added for Site Modules, Administrator modules and Custom Administrator Menus when using the alternative preset.

Quickstarts are added for Administrator modules and Custom Administrator Menus when using the default preset.

### Testing Instructions
Create a custom Administrator menu
Display the default Preset menu in the Admin menu module.
Then switch to the Alternative preset.

No  + sign (quickstart) for the items concerned

patch and test again. 

test clicking the + sign to confirm it does what it should.

### After patch
for both presets

For the custom administrator menu:
![Screen Shot 2019-11-11 at 09 50 58](https://user-images.githubusercontent.com/869724/68573844-e008bb80-0468-11ea-87fd-e7245e42e1ed.png)

For the administrator modules
![Screen Shot 2019-11-11 at 09 52 27](https://user-images.githubusercontent.com/869724/68573972-2bbb6500-0469-11ea-8096-cdf02393d760.png)

For the alternate preset is also added the quickstart for site modules
![Screen Shot 2019-11-11 at 09 54 49](https://user-images.githubusercontent.com/869724/68574045-56a5b900-0469-11ea-955c-dd59b5df13e3.png)
